src/modules/songs/components/SongTagSelector/SongTagSelector.tsx

```
@@ -0,0 +1,407 @@
+                          "mx-1 flex items-center justify-between rounded-lg px-3 py-2 text-sm transition-colors",
+                          isSelected
+                            ? "cursor-default opacity-90"
+                            : "cursor-pointer",
+                          highlightedIndex === index && !isSelected
+                            ? "bg-primary text-primary-foreground hover:bg-primary/90"
+                            : !isSelected
+                              ? "hover:bg-secondary/50"
+                              : "",
+                        )}
+                        data-index={!isSelected ? index : undefined}
+                      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)